src/vs/workbench/contrib/chat/common/chatServiceImpl.ts

```diff
@@ -609,11 +609,11 @@ export class ChatService extends Disposable implements IChatService {
 		return undefined;
 	}
 
-	loadSessionFromContent(data: IExportableChatData | ISerializableChatData): IChatModel | undefined {
-		return this._startSession(data, data.initialLocation ?? ChatAgentLocation.Panel, true, CancellationToken.None);
+	loadSessionFromContent(data: IExportableChatData | ISerializableChatData, inputType: string = ''): IChatModel | undefined {
```

Comment:
And this default we should be able to remove

Reply:
@roblourens good catch, there were a lot of more places where we needed to remove this